### PR TITLE
Fewer Mid-Round Events

### PR DESCRIPTION
**Pull request by DEATHB4DEFEAT**
# Description

Requires https://github.com/Simple-Station/Parkstation-Friendly-Chainsaw/pull/58

# Changelog

:cl:
- tweak: Mid-round events will occur way less often

### CODE REVIEW

File: Content.Server/StationEvents/BasicStationEventSchedulerSystem.cs [hunk 1]
```diff
@@ -57,8 +57,8 @@ public override void Update(float frameTime)
         /// </summary>
         private void ResetTimer(BasicStationEventSchedulerComponent component)
         {
-            // 5 - 25 minutes. TG does 3-10 but that's pretty frequent
-            component.TimeUntilNextEvent = _random.Next(300, 1500);
+            // 15-35 minutes
+            component.TimeUntilNextEvent = _random.Next(900, 2100); // Parkstation-SlowEvents
```
Comment (Pspritechologist) [inline Content.Server/StationEvents/BasicStationEventSchedulerSystem.cs:61]: This fucking sucks, I can't believe this works like this.

https://github.com/space-wizards/space-station-14/pull/25959

File: Content.Server/StationEvents/BasicStationEventSchedulerSystem.cs [hunk 2]
```diff
@@ -57,8 +57,8 @@ public override void Update(float frameTime)
         /// </summary>
         private void ResetTimer(BasicStationEventSchedulerComponent component)
         {
-            // 5 - 25 minutes. TG does 3-10 but that's pretty frequent
-            component.TimeUntilNextEvent = _random.Next(300, 1500);
+            // 15-35 minutes
```
Comment (Pspritechologist) [inline Content.Server/StationEvents/BasicStationEventSchedulerSystem.cs:60]: 15-35 seems like it may be a little high. I'll leave it to other opinions.

Comment (DEATHB4DEFEAT) [reply Content.Server/StationEvents/BasicStationEventSchedulerSystem.cs:60]: That's how it was previously

**Review by Pspritechologist — APPROVED** [review]
Annotate your comment changes :trollface: 

Also yeah, label this as DNM if you want to wait to see the results of the pr. Not like anyone is playing with it anytime soon :P

**Comment by DEATHB4DEFEAT** [comment]
Do you have any idea what's going on with the test fails?

**Comment by Pspritechologist** [comment]
Looks like the WizDen pr should get merged quickly enough, probably worth waiting.

**Comment by DEATHB4DEFEAT** [comment]
https://github.com/Simple-Station/Einstein-Engines/pull/486

**Pull request closed without merging**